console2/src/api/service/console/user/index.ts

```diff
@@ -18,30 +18,34 @@
  * =====
  */
 
-import { ConcordKey, fetchJson, queryParams } from '../../../common';
-import { ProcessEntry, ProcessStatus } from '../../../process';
-
-export interface ProjectProcesses {
-    projectName: ConcordKey;
-    running: number;
-}
+import {ConcordId, ConcordKey, fetchJson, queryParams} from '../../../common';
+import { ProcessEntry } from '../../../process';
 
 export interface UserActivity {
-    processStats: {
-        status: ProcessStatus;
-        count: number;
-    };
-    orgProcesses?: {
-        orgName: ConcordKey;
-        processes: ProjectProcesses[];
-    };
     processes: ProcessEntry[];
 }
 
+export interface ProcessCardEntry {
+    id: ConcordId;
+    orgName: ConcordKey;
+    projectName: ConcordKey;
+    repoName: ConcordKey;
+    entryPoint: string;
+    name: string;
+    description?: string;
+    icon?: string;
+    isCustomForm: boolean;
+}
+
 export const getActivity = (
-    maxProjectsPerOrg: number,
     maxOwnProcesses: number
 ): Promise<UserActivity> =>
     fetchJson(
-        `/api/service/console/user/activity?${queryParams({ maxProjectsPerOrg, maxOwnProcesses })}`
+        `/api/v2/service/console/user/activity?${queryParams({ maxOwnProcesses })}`
     );
+
+export const listProcessCards = (
+): Promise<ProcessCardEntry[]> =>
+    fetchJson(
+        `/api/v2/service/console/user/process-card`
```

Comment:
Should it be `/api/v1/org/process-card` instead?
On the same note, I think we should use `/api/v1/processCard` prefix instead of putting it under `/api/v1/org` (the next URL part is taken by `{orgId}` anyway).